ext/fs/nffs/Kconfig

```diff
@@ -0,0 +1,17 @@
+
+config NFFS_FILESYSTEM
+	bool "NFFS File System"
+	depends on FILE_SYSTEM_NFFS
+	default n
+	help
+	Use the NFFS File system implementation.
+
+config NFFS_FILESYSTEM_MAX_AREAS
```

Comment:
can we have a help here and below. What is it? How these values impose on the file system(performance, corner cases etc.).